lib/internal/per_context/domexception.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+const {
+  SafeWeakMap,
+  SafeMap,
+  Object,
+  Symbol
+} = primordials;
+
 class ERR_INVALID_THIS extends TypeError {
```

Comment:
On a side note, we'd probably want to make the Errors part of primordials too, but I am not yet sure how this would work for e.g. error-serdes, or if how it works with our internal errors (because we do need to assign properties to the instances dynamically, they probably can't be frozen).